### PR TITLE
feat(VerticalMenu): Added a vertical menu component

### PR DESCRIPTION
**Pull request by andrewleyva**
Initial Simple Version of Vertical Menu

### CODE REVIEW

**Comment by DavidWhitlockTripwire** [comment]
@andrewleyva and @cdaringe, I'm totally ignorant here: Are we expecting the circleci tests for fail with these changes?

**Comment by andrewleyva** [comment]
It failed because I didn't push up everything... oops.

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewwtejblnhqtfljma.surge.sh](http://surgereviewwtejblnhqtfljma.surge.sh)

**Comment by andrewleyva** [comment]
Uhhhhhhh... I need to convert this to use React Router I think...
Because that looks like a better way to do this.

File: src/components/native/VerticalMenu/vertical-menu.examples.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Vertical Menu
+
+```js
+
+<VMenu test="xxx">
```
Comment (DavidWhitlockTripwire) [inline src/components/native/VerticalMenu/vertical-menu.examples.md]: Is this "test" attribute necessary?  I don't know anything about this.

File: src/components/native/VerticalMenu/vertical-menu.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <ul className='vmenu__content'>{this.props.children}</ul>
+      </nav>
+    )
+  }
+}
+
+VMenu.MenuItem = VMenuItem
+
+VMenu.propTypes = {
+  /** One of: MenuItem, Node */
```
Comment (DavidWhitlockTripwire) [inline src/components/native/VerticalMenu/vertical-menu.jsx]: Capital "N" "Node" or lowercase?  Line 38 has lowercase.

File: src/components/native/VerticalMenu/vertical-menu.examples.md (hunk 1 above)
Comment (andrewleyva) [reply src/components/native/VerticalMenu/vertical-menu.examples.md]: Oops, didn't double check that example before committing. That's unnecessary.

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewklealejmlyclfll.surge.sh](http://surgereviewklealejmlyclfll.surge.sh)

**Comment by andrewleyva** [comment]
Thanks, looking into it and I think you're right.

**Comment by antoinejuhel** [comment]
Hello @cdaringe,

- I don't have a particular affinity for one or the other. I simply reused what Connect was already using to avoid creating yet another difference between the two UIs. I will say that the rounded corners of the menu provided by the framework doesn't lend itself as well to our current aesthetic compared to our current one.
- Additionally, it seems Yung should also be part of these discussions so I will ask her if she can create a GitHub account to make these convos a bit more seem-less.

**Comment by andrewleyva** [comment]
Hey @antoinejuhel,
I think @cdaringe's comment was regarding the underlying code. We still want it to look like the menu in TOC and Connect.

**Comment by antoinejuhel** [comment]
@andrewleyva Got it.

**Pull request closed without merging**